x-pack/plugins/lens/public/indexpattern_datasource/operations/definitions/formula/util.ts

```diff
@@ -548,11 +576,170 @@ Returns a default numeric value when value is null.
+Performs an equality comparison between two values.
+To be used as condition for \`ifelse\` comparison function.
+Also works with \`==\` symbol.
+
+Example: Returns true if the average of bytes is exactly the same amount of average memory
+\`average(bytes) == average(memory)\`
+
+Example: \`eq(sum(bytes), 1000000)\`
+    `,
+    }),
+  },
+  lte: {
+    section: 'comparison',
+    positionalArguments: [
+      {
+        name: i18n.translate('xpack.lens.formula.left', { defaultMessage: 'left' }),
+        type: getTypeI18n('number'),
+      },
+      {
+        name: i18n.translate('xpack.lens.formula.right', { defaultMessage: 'right' }),
+        type: getTypeI18n('number'),
+      },
+    ],
+    outputType: getTypeI18n('boolean'),
+    help: i18n.translate('xpack.lens.formula.lteFunction.markdown', {
+      defaultMessage: `
+Performs a lower than or equal comparison between two values.
+To be used as condition for \`ifelse\` comparison function.
+Also works with \`<=\` symbol.
+
+Example: Returns true if the average of bytes is lower than or equal to the average amount of memory
+\`average(bytes) <= average(memory)\`
+
+Example: \`lte(average(bytes), 1000)\`
+    `,
+    }),
+  },
+  gte: {
+    section: 'comparison',
+    positionalArguments: [
+      {
+        name: i18n.translate('xpack.lens.formula.left', { defaultMessage: 'left' }),
+        type: getTypeI18n('number'),
+      },
+      {
+        name: i18n.translate('xpack.lens.formula.right', { defaultMessage: 'right' }),
+        type: getTypeI18n('number'),
+      },
+    ],
+    outputType: getTypeI18n('boolean'),
+    help: i18n.translate('xpack.lens.formula.gteFunction.markdown', {
+      defaultMessage: `
+Performs a greater than comparison between two values.
+To be used as condition for \`ifelse\` comparison function.
+Also works with \`>=\` symbol.
+
+Example: Returns true if the average of bytes is greater than or equal to the average amount of memory
+\`average(bytes) >= average(memory)\`
+
+Example: \`gte(average(bytes), 1000)\`
+    `,
+    }),
+  },
+  ifelse: {
+    section: 'comparison',
+    positionalArguments: [
+      {
+        name: i18n.translate('xpack.lens.formula.condition', { defaultMessage: 'condition' }),
+        type: getTypeI18n('boolean'),
+      },
+      {
+        name: i18n.translate('xpack.lens.formula.left', { defaultMessage: 'left' }),
+        type: getTypeI18n('number'),
+      },
+      {
+        name: i18n.translate('xpack.lens.formula.right', { defaultMessage: 'right' }),
+        type: getTypeI18n('number'),
+      },
+    ],
+    help: i18n.translate('xpack.lens.formula.ifElseFunction.markdown', {
+      defaultMessage: `
+Returns a value depending on whether the element of condition is true or false.
+
+Example: Average revenue per customer but in some cases customer id is not provided which counts as additional customer
+\`sum(total)/(cardinality(customer_id) + ifelse(count() > count(filter=customer_id:*), 1, 0)\`
```

Comment:
this isn't valid formula:
```
sum(total)/(unique_count(customer_id) + ifelse(count() > count(kql='customer_id:*'), 1, 0)
```

Reply:
it's not just the missing closing bracket, it's also cardinality->unique_count and filter->kql and quotes around the filter

Reply:
All fixed and tested.